src/ec/packages/ec-component-select/select.html.twig

```diff
@@ -60,16 +66,34 @@
 <div class="ecl-form-group ecl-form-group--select">
   {% if _label is not empty %}
     <label
-      class="ecl-form-label{{ invalid ? ' ecl-form-label--invalid' }}{{ hide_label ? ' ecl-form-label--hidden'}}"
-      for="{{ _id }}"
+      {{ _id ? 'for="' ~ _id ~  '"' }}
```

Comment:
mind that you have two spaces here: `_id ~  '"'`

Reply:
Thx, done.